src/net_processing.cpp

```diff
@@ -184,6 +201,8 @@ static constexpr double MAX_ADDR_RATE_PER_SECOND{0.1};
 static constexpr size_t MAX_ADDR_PROCESSING_TOKEN_BUCKET{MAX_ADDR_TO_SEND};
 /** The compactblocks version we support. See BIP 152. */
 static constexpr uint64_t CMPCTBLOCKS_VERSION{2};
+/** Used to determine whether to fanout or to reconcile a transaction with a given peer */
```

Comment:
could say “low-fanout flooding” as it’s the term used above to describe the two transaction announcement strategy